src/features/views/components/ViewDataTable/columnTypes/PersonTagColumnType/index.tsx

```diff
@@ -57,14 +57,18 @@ export default class PersonTagColumnType implements IColumnType {
         apiClient.get<ZetkinTag>(`/api/orgs/${orgId}/people/tags/${tagId}`),
     });
 
-    const tag = tagFuture.data;
+    const tag = !accessLevel ? tagFuture.data : null;
```

Comment:
I think the `loadItemIfNecessary()` party should also be conditional, so that we don't try to load something that we know that we won't be able to retrieve (and also won't use).

Reply:
Makes perfect sense :)